### PR TITLE
Announce Scala 2.12.3

### PR DESCRIPTION
**Pull request by retronym**
[not for merge until the release is performed]

### CODE REVIEW

File: news/_posts/2017-07-26-release-notes-2.12.3.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+layout: news
+post-type: announcement
+permalink: /news/2.12.3
+title: "Scala 2.12.3 is now available!"
+---
+## Changes
+
+- Compiler performance has [improved significantly](developer.lightbend.com/blog/2017-06-12-faster-scala-compiler/) and is now [automatically benchmarked](https://scala-ci.typesafe.com/grafana/) after each change.
```
Comment (godenji) [inline news/_posts/2017-07-26-release-notes-2.12.3.md]: needs absolute link here or else 404s.

**Comment by heathermiller** [comment]
What's the status of this? Seems like the release went out yesterday, no? Can we merge this?

**Comment by retronym** [comment]
Release is staged, but not yet pushed as I was travelling. Will finish the
release today and merge this once it is on maven central.

**Comment by retronym** [comment]
This still needs a bit of reshuffling after the website refactor. AFAICT, `news/_posts/*` is the old location and we should now direct the posts into `_posts`.

**Pull request merged**